pkg/bindinfo/binding_cache.go

```diff
@@ -52,26 +56,42 @@ type fuzzyBindingCache struct {
 	fuzzy2SQLDigests map[string][]string // fuzzyDigest --> sqlDigests
 
 	sql2FuzzyDigest map[string]string // sqlDigest --> fuzzyDigest
+
+	loadBindingFromStorageFunc func(sqlDigest string) (Bindings, error)
```

Comment:
please add some comments for this function.

Reply:
updated